plugins/web-terminal/src/components/NamespacePickerDialog/index.tsx

```diff
@@ -0,0 +1 @@
+export { NamespacePickerDialog as NamespaceModalComponent } from './NamespacePickerDialog';
```

Comment:
This kept the old name (as well as the import in the `TerminalComponent`). Please change.